docs/CLI.md

```diff
@@ -298,7 +298,21 @@ evergreen --delete-tag KEY
 ```
 Note these tags cannot overwrite Evergreen tags. 
 
-Hosts can be set to never expire using the `--no-expire` tag (although each user has a limit for these kinds of hosts). Hosts can be set to expire again using the `--expire` tag, which will set the host to expire in 24 hours. This expiration can be extended using `--extend <hours>`, and you can extend its lifetime up to a max of 30 days past host creation.
+Hosts can be set to never expire using the `--no-expire` option. Keep in mind that if making a host unexpirable from the
+CLI, you should also set up a [sleep schedule](Hosts/Spawn-Hosts#unexpirable-host-sleep-schedules) from the command
+line as well; if you don't set one, your unexpirable host will be automatically assigned a default sleep schedule. The
+sleep schedule can be set in the CLI request. To set daily times when you'd like your host to be running/stopped, set
```

Comment:
"The sleep schedule can be set in the CLI request" sounds like it was just said before when it said "from the command line as well", unless it's trying to convey something else. Does the help command show the flag options nicely in our cli? So maybe we could direct them to do `evergreen host create --help` or what not for help. It might be better if we want to make the docs more ambiguous and rely on our CLI output as documentation but might make it more difficult for users to adapt.

Reply:
I only wrote these docs mostly for completeness because a lot of the CLI docs restate what you could see in `--help` text. But honestly I'm fine with just not having them since users should be familiar with CLIs having help text. Left the example in just in case they need a reference.